addons/web/static/src/js/chrome/action_manager_act_window.js

```diff
@@ -689,7 +689,7 @@ ActionManager.include({
                 };
```

Comment:
commit message (suggestion):

[FIX] web: data-mobile on action_manager_act_window

Commit ca4eaf4 introduced the data-mobile attribute, used
to specify options to pass when executing an action, only in
mobile (like using a fullscreen modal).

However, we forgot to add the 'isMobile' check, so the option
was passed in desktop as well.

This commit fixes the issue and adds some tests.

Task ID: 2155601